src/sentry/backup/comparators.py

```diff
@@ -806,6 +806,10 @@ def get_default_comparators() -> dict[str, list[JSONScrubbingComparator]]:
                 DateUpdatedComparator("date_added", "date_updated"),
             ],
             "sentry.groupsearchview": [DateUpdatedComparator("date_updated")],
+            "sentry.groupsearchviewproject": [
+                DateUpdatedComparator("date_updated"),
+                DateUpdatedComparator("date_added"),
+            ],
```

Comment:
Hmm, this is annoying that we still need to do it sometimes... I'll try look into fixing it